package.json

```
@@ -96,7 +96,7 @@
     "puppeteer": "^9.1.1",
     "react-refresh": "^0.10.0",
     "sourcemap-validator": "^2.1.0",
-    "type-fest": "^1.2.1",
+    "type-fest": "^1.2.1 || ^2.0.0",
```

`^2.0.0` doesn't work - particularly because it dropped support for Node 10.x.

In reality we could accept it, but this will fail tests and I don't think it is ensured to work due to Sindre's move to ESM.

```suggestion
    "type-fest": "^1.2.1",
```